proto/process/agent.proto

```diff
@@ -570,35 +578,134 @@ message ServiceSessionAffinityConfig {
     int32 clientIPTimeoutSeconds = 1;
 }
 
+// reference https://github.com/kubernetes/kubernetes/blob/cb19b56831d54d1d31249949318ef0b07bf00df9/staging/src/k8s.io/api/core/v1/generated.proto#L2107
+message Node {
+	Metadata metadata = 1;
+	NodeSpec spec = 2;
+	NodeStatus status = 3;
```

Comment:
add tag field

Reply:
https://github.com/DataDog/agent-payload/blob/bb640080328f5a9b22f3f650afca01f6ef1000b0/proto/process/agent.proto#L766-L767

Reply:
We should add tags and host just like for Pod. These won't be filled by the agent, but by the orchestrator-resolver, which must resolve the host for each node in the message with HMS, and attach tags and the host object.